Make.inc

```diff
@@ -1018,6 +1018,10 @@ PATCHELF := patchelf
 else
 PATCHELF := $(build_depsbindir)/patchelf
 endif
+# In the standard build system we want to patch files with `--set-rpath`, but downstream
+# packagers like Spack may want to use `--add-rpath` instead, leave them the possibility to
+# choose the command.
+PATCHELF_SET_RPATH := --set-rpath
```

Comment:
Can we call it PATCHELF_SET_RPATH_ARG? Otherwise it sounds like it should be a Boolean.

Reply:
Sounds good.  Done